local_sales/app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag "application", "data-turbo-track": "reload" %>
```

Comment:
I remember that TurboLinks is also called here to verify it.

```suggestion
    <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Reply:
you're right